coalib/results/Result.py

```diff
@@ -60,8 +60,8 @@ def __init__(self,
             A message which may help the user find out why this result was
```

Comment:
Could you maybe leave out the "definition" of "docstring definition" in your commit message?
Actually we have a component in coala called `DocstyleDefinition` which is highly related to this term and defines a set of properties that identify a docstring in code (like the delimiters, in python these would be `"""`).